extensions/2.0/Khronos/KHR_materials_sheen/README.md

```diff
@@ -0,0 +1,237 @@
+* Mike Bond, Adobe, [@MiiBond](https://github.com/MiiBond)
+* Don McCurdy, Google, [@donmccurdy](https://twitter.com/donrmccurdy)
+* Gary Hsu, Microsoft, [@bghgary](https://twitter.com/bghgary)
+* Jekfer Bichon, Dassault System, [@EliphasNUIT](https://github.com/EliphasNUIT)
+* Aidin Abedi, Epic Games, [@aidinabedi](https://github.com/aidinabedi)
+
+Copyright (C) 2018-2020 The Khronos Group Inc. All Rights Reserved. glTF is a trademark of The Khronos Group Inc.
+See [Appendix](#appendix-full-khronos-copyright-statement) for full Khronos Copyright Statement.
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Exclusions
+* This extension must not be used on a material that also uses `KHR_materials_pbrSpecularGlossiness`.
+* This extension must not be used on a material that also uses `KHR_materials_unlit`.
+
+## Overview
+
+This extension defines a sheen that can be layered on top of an existing glTF material definition. A sheen layer is a common technique used in Physically-Based Rendering to represent cloth and fabric materials, for example. See [Theory, Documentation and Implementations](#theory-documentation-and-implementations)
+
+## Extending Materials
+
+The PBR sheen materials are defined by adding the `KHR_materials_sheen` extension to any compatible glTF material (excluding those listed above). 
+For example, the following defines a material like velvet.
+
+```json
+{
+    "materials": [
+        {
+            "name": "velvet",
+            "extensions": {
+                "KHR_materials_sheen": {
+                    "sheenColorFactor": [0.9, 0.9, 0.9]
+                }
+            }
+        }
+    ]
+}
+```
+
+### Sheen
+
+The following parameters are contributed by the `KHR_materials_sheen` extension:
+
+|                                  | Type                                                                            | Description                            | Required                       |
+|----------------------------------|---------------------------------------------------------------------------------|----------------------------------------|--------------------------------|
+|**sheenColorFactor**                   | `array`                                                                         | The sheen color in linear space        | No, default: `[0.0, 0.0, 0.0]` |
+|**sheenColorTexture**         | [`textureInfo`](/specification/2.0/README.md#reference-textureInfo)             | The sheen color (RGB).<br> The sheen color is in sRGB transfer function | No               |
+|**sheenRoughnessFactor**               | `number`                                                                        | The sheen roughness.                   | No, default: `0.0`             |
+|**sheenRoughnessTexture**         | [`textureInfo`](/specification/2.0/README.md#reference-textureInfo)             | The sheen roughness (Alpha) texture. | No               |
+
```

Comment:
I guess to stay consistent with the core spec - for instance pbrmetallicroughness:
https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#reference-pbrmetallicroughness

Then we should declare usage of each of the properties, for instance what texture component(s) the value is sampled from_

### sheen.sheenRoughnessTexture 
The sheen roughess value, roughness is sampled from the A channel.

etc etc

Reply:
I tried to stay consistent with the other extensions: https://github.com/KhronosGroup/glTF/blob/master/extensions/2.0/Khronos/KHR_materials_clearcoat/README.md

Should they be reviewed as well then ?

Reply:
These are mentioned in the schema (e.g. https://github.com/KhronosGroup/glTF/pull/1688/files#diff-30c33ab0604a6cc121306aa2e409155dR23) which is kind of the equivalent of the core spec since the properties reference is generated from the schema.

Reply:
Ok thanks @bghgary  could you elaborate what that means in this particular case?
Do we need to change anything in this extension to be consistent with the core spec in the future?